tests/test_calculations.py

```diff
@@ -19,6 +19,14 @@ def test_gc_content_3(): #Reference: https://jamiemcgowan.ie/bioinf/gc_content.h
     oprimer = Primer("ATTTTTT")
     assert oprimer.gc_content == 0
 
+def test_gc_clamp_1(): #Reference: https://www.oligoevaluator.com/OligoCalcServlet
+    oprimer = Primer("ATCGATCGATCGATCGATCG")
```

Comment:
- Use this sequence: `ATCGATCGATCGATCGGTCG`
- Use this website: https://www.bioinformatics.org/sms2/pcr_primer_stats.html

Reply:
It does not give me the `gc_clamp` value:
![Screenshot from 2024-12-25 10-04-25](https://github.com/user-attachments/assets/b7dcebf9-d88a-43ab-821e-c8c45a94e831)